src/pages/SwapPage/layout/Checkout/SwapForm.tsx

```diff
@@ -0,0 +1,209 @@
+import * as React from 'react';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import TextField from '@mui/material/TextField';
+import { Button } from '@mui/material';
+import SwapHorizIcon from '@mui/icons-material/SwapHoriz';
+import ButtonNew from 'components/common/ButtonNew';
+import styles from './styles.module.scss';
+
+interface IProps {
+	handleSetFromToken: (arg: string) => void;
+	handleSetToToken: (arg: string) => void;
+	ApproveSwapTokens: () => void;
+	SwapTokens: () => void;
+	handleSetAmountIn: (arg: string) => void;
+	handleSetMinAmountOut: (arg: string) => void;
+	fromSupertoken: string;
+	toSupertoken: string;
+	amountIn: string;
+	minAmountOut: string;
+	tokens: { name: string; address: string; underlyingToken: string }[];
+	approved: boolean;
+	isLoading: boolean;
+}
+
+export const SwapForm: React.FC<IProps> = ({
+	tokens,
+	ApproveSwapTokens,
+	SwapTokens,
+	handleSetFromToken,
+	handleSetToToken,
+	handleSetAmountIn,
+	handleSetMinAmountOut,
+	fromSupertoken,
+	toSupertoken,
+	amountIn,
+	minAmountOut,
+	approved,
+	isLoading,
+}) => {
+	return (
+		<React.Fragment>
+			<Grid
+				container
+				spacing={3}
+				style={{
+					backgroundColor: '#2d3233',
+				}}
+			>
+				<Grid item xs={12} sm={12}>
+					<Typography
+						variant="h6"
+						gutterBottom
+						style={{
+							color: 'white',
+							fontSize: 'larger',
+						}}
+					>
+						From Supertoken
```

Comment:
Do not use the word "Supertoken" just use "Token," we will hide all traces superfluid from the app so that users don't even know that Ricoeht uses superfluid